libs/blocks/marketo/marketo.js

```diff
@@ -76,11 +76,16 @@ export const formSuccess = (form) => {
 const readyForm = (form) => {
   const formEl = form.getFormElem().get(0);
 
-  formEl.addEventListener('focus', (e) => {
-    /* c8 ignore next 5 */
-    if (!['text', 'email', 'tel', 'textarea'].includes(e.target.type)) return;
+  formEl.addEventListener('focus', ({ target }) => {
+    /* c8 ignore next 10 */
+    const isDesktop = window.innerWidth > 900;
```

Comment:
That's a good thought, but I deliberately have it inside the focus function so that the isDesktop variable will be accurate even if the user resizes the window.

Reply:
Same would be true for the `matchMedia` approach, you would just use `isDesktop.matches` and it will reflect the current state of the viewport.

Reply:
Yeah I see what you're saying now. I've updated it